models/staging/ga4/stg_ga4__items.sql

```diff
@@ -0,0 +1,42 @@
+{{
```

Comment:
@dgitis if you look at `main`, this was renamed to `stg_ga4_event_items.sql` to better describe that this model contains event items (pk is event_key), as opposed to a model of items (pk is item_id).

Can you delete and refactor?